hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/multipart/S3MultipartUploadCompleteRequest.java

```diff
@@ -499,20 +499,20 @@ private long getMultipartDataSize(String requestedVolume,
     for (OzoneManagerProtocolProtos.Part part : partsList) {
       currentPartCount++;
       int partNumber = part.getPartNumber();
-      String partName = part.getPartName();
+      String eTag = part.getETag();
```

Comment:
It might make sense to check if the request has eTags in it, if the S3G is an old client, it might not have the updated client side logic.